sipa/static/js/hints.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "not_in": [],
+    "patterns": [
+      "forgot.{0,10}password",
+      "can.+t.{0,10}login",
+      "reset.{0,10}password",
+      "passwort.{0,10}vergessen",
+      "kann.{0,10}nicht.{0,10}einloggen",
+      "passwort.{0,10}zurücksetzen"
+    ],
+    "hint_de": "Wir können dein Passwort nicht zurücksetzen ohne dich zu identifizieren. Bitte komme mit einem Lichtbildausweis zu einer unserer <a href='pages/support/contacts'>Sprechstunden</a>",
```

Comment:
Use a `hints` key with an object as value, whose keys are locales:

```json
"hints": {
    "de": "Wir können dein Passwort nicht zurücksetzen ohne dich zu identifizieren. Bitte komme mit einem Lichtbildausweis zu einer unserer <a href='pages/support/contacts'>Sprechstunden</a>", 
    "en": "We can't reset your password without identifying you first. Please come with an id card to one of our <a href='pages/support/contacts'>office hours</a>"
}

Reply:
done in a8a7ee8